cfgov/v1/__init__.py

```diff
@@ -11,6 +11,7 @@
 def environment(**options):
     options.setdefault('extensions', []).append(CompressorExtension)
     env = sheerlike_environment(**options)
+    env.autoescape = True
```

Comment:
one of the security findings was "jinja2 sets autoescape to False." So making sure the env explicitly sets this